src/config.rs

```diff
@@ -27,6 +27,9 @@ pub(crate) struct Link {
 
     /// How often sending a message works vs. the message getting dropped
     pub(crate) message_loss: Option<MessageLoss>,
+
+    /// How often a sent message is duplicated. Only applies to UDP.
+    pub(crate) message_duplication: Option<MessageDupliaction>,
```

Comment:
Typo here. Should be `duplication` not `dupliaction`

Reply:
The struct name also has the typo